Application/Api/Reporting/Univention/Univention.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace SPHERE\Application\Api\Reporting\Univention;
+
+use MOC\V\Core\FileSystem\FileSystem;
+use SPHERE\Application\IModuleInterface;
+use SPHERE\Application\IServiceInterface;
+use SPHERE\Application\Setting\Authorization\Account\Account;
+use SPHERE\Common\Frontend\IFrontendInterface;
+use SPHERE\Common\Main;
+
+/**
+ * Class Custom
+ *
+ * @package SPHERE\Application\Api\Reporting\Univention
+ */
+class Univention implements IModuleInterface
+{
+
+    public static function registerModule()
+    {
+
+        Main::getDispatcher()->registerRoute(Main::getDispatcher()->createRoute(
+            __NAMESPACE__.'/SchoolList/Download',
+            __NAMESPACE__.'/Univention::downloadSchoolList'
+        ));
+        Main::getDispatcher()->registerRoute(Main::getDispatcher()->createRoute(
+            __NAMESPACE__.'/User/Download',
+            __NAMESPACE__.'/Univention::downloadUserList'
+        ));
+
```

Comment:
Codacy found an issue: [Functions must not contain multiple empty lines in a row; found 2 empty lines](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)